### DISCUSSION TITLE
Any plans for XLIFF 2.1 support?

### DISCUSSION THREAD

**Discussion by davidarndtsick** [maxprograms-com/OpenXLIFF#23] [Ideas]
Hi, 
I just wanted to ask, if there were already any plans for supporting XLIFF 2.1 validation & conversion.
Shouldn't be a huge step from XXLIFF 2.0, as I understand it.

Best regards
David

**Comment by rmraya** [comment]
Yes, validation of XLIFF 2.1 and the upcoming 2.2 will be added soon.

**Reply by davidarndtsick** [reply]
cool, thanks!

**Comment by rmraya** [comment]
Implemented in current code
